draft-irtf-cfrg-vdaf.md

```diff
@@ -3389,8 +3453,8 @@ throws an error.
 ## Instantiation {#poplar1-inst}
 
 By default, Poplar1 is instantiated with IdpfPoplar (`VALUE_LEN == 2`) and
-PrgSha3 ({{prg-sha3}}). This VDAF is suitable for any positive value of `BITS`.
-Test vectors can be found in {{test-vectors}}.
+PrgFixedKeyAes128 ({{prg-fixed-key-aes-128}}). This VDAF is suitable for any
+positive value of `BITS`. Test vectors can be found in {{test-vectors}}.
```

Comment:
This should be clarified to say that Poplar1 is instantiated with PrgSha3 for the VDAF itself, and the IdpfPoplar inside is instantiated with PrgFixedKeyAes128.

Reply:
Replaced the reference here with PrgSha3.